docs/src/pages/landing-page/Index.vue

```diff
@@ -134,13 +134,55 @@
           <div class="lp-heading lp-heading-large">
             What our community thinks of quasar
           </div>
-          <div class="text-size-20 text-lp-light text-weight-medium">
+          <div class="lp-heading lp-heading-small">
```

Comment:
I'd use use BEM style `lp-heading--small` for the modifier class here and with similar classes